features/nanostack/mbed-mesh-api/source/WisunBorderRouter.cpp

```diff
@@ -324,3 +341,62 @@ mesh_error_t WisunBorderRouter::get_radius_shared_secret(uint16_t *shared_secret
 
     return MESH_ERROR_NONE;
 }
+
+mesh_error_t WisunBorderRouter::set_radius_timing(ws_br_radius_timing_t *timing)
+{
+    if (timing == NULL) {
+        return MESH_ERROR_PARAM;
+    }
+
+    if (validate_radius_timing(timing) != MESH_ERROR_NONE) {
+        return MESH_ERROR_PARAM;
+    }
+
+    _radius_timing = *timing;
+    _radius_timing_set = true;
+
+    return set_bbr_radius_timing();
+}
+
+mesh_error_t WisunBorderRouter::set_bbr_radius_timing(void)
+{
+    int status = ws_bbr_radius_timing_set(_mesh_if_id, (bbr_radius_timing_t *) &_radius_timing);
```

Comment:
If this structure is modified having the same structure in both interface makes changing of it quite hard

Reply:
Yes, true. I removed the dependency.